docs/components/connectors/connector-types.md

```diff
@@ -14,13 +14,29 @@ Looking for pre-built, [Out-of-the-box Connectors](/components/connectors/out-of
 
 Outbound Connectors allow workflows to trigger external systems or services, making it possible to integrate workflows with other parts of a business process or system architecture.
 
+The programming code in Java to connect to the external system is executed when the workflow reaches the service task.
```

Comment:
This sentence is a little awkward. Is there any reason we can't refer to this as "The Java code" like we do a few lines below?

Reply:
```suggestion
The Java code to connect to the external system is executed when the workflow reaches the service task.
```

Reply:
I believe it's taken verbatim from [here](https://github.com/camunda/camunda-platform-docs/blob/b75513e4c095bc62dc54fe0291edab4a2f79c985/docs/components/connectors/introduction.md?plain=1#L24). Corrected it here, doesn't look good to me either 🙂

Reply:
Always room for improvement 😆 thank you!